make-release.sh

```diff
@@ -39,7 +39,7 @@ if [[ ! ${VERSION} ]]; then
   exit 1
 fi
 
-if [[ ${USE_TMP_DIR} ]]; then
+if [[ ${USE_TMP_DIR} -eq 1 ]]; then
```

Comment:
this will never be true. Check out line 18, where you're changing it from 0 to 0 :|